server/src/serializers/IngredientSerializer.js

```diff
@@ -9,6 +9,26 @@ class IngredientSerializer {
 
     return serializedIngredient
   }
+
+  static async getIngredientWithPantryMeasurements(ingredient) {
+    const allowedAttributes = ["name", "id"]
+
+    let serializedIngredient = {}
+    allowedAttributes.forEach(attribute => {
+      serializedIngredient[attribute] = ingredient[attribute]
+    })
+
+    const pantryMeasurement = await ingredient.$relatedQuery("pantryMeasurement").first()
```

Comment:
hmmm, is it true that we will always want the first pantryMeasurement that is related to this ingredient for this endpoint? This seems like an odd data set to get. I could more easily see you grabbing the related pantryMeasurement if we were on some "Pantry Ingredients List" that shows all of the ingredients in one user's pantry (a different feature to be sure)